src/qibo/quantum_info/random_ensembles.py

```diff
@@ -426,25 +419,13 @@ def random_statevector(dims: int, haar: bool = False, seed=None, backend=None):
             TypeError, "seed must be either type int or numpy.random.Generator."
         )
 
-    if backend is None:  # pragma: no cover
-        backend = GlobalBackend()
-
-    local_state = (
-        np.random.default_rng(seed) if seed is None or isinstance(seed, int) else seed
-    )
+    backend, local_state = _set_backend_and_local_state(seed, backend)
 
-    if not haar:
-        # sample real and imag parts of complex amplitude in [-1, 1]
-        state = 1j * (2 * local_state.random(dims) - 1)
-        state += 2 * local_state.random(dims) - 1
-        state /= np.linalg.norm(state)
-        state = backend.cast(state, dtype=state.dtype)
-    else:
-        # select a random column of a haar random unitary
-        k = local_state.integers(low=0, high=dims)
-        state = random_unitary(dims, measure="haar", seed=seed, backend=backend)[:, k]
+    state = local_state.standard_normal(dims).astype(complex)
+    state += 1.0j * local_state.standard_normal(dims)
+    state /= np.linalg.norm(state)
```

Comment:
```suggestion
    state /= np.linalg.norm(state)
    state =  backend.cast(state, dtype=state.dtype)
```

Reply:
This is not important but in the rest of the functions of the module, a variable `x = backend.cast()` is defined and then `x` is returned. This way we are consistent.